website/blog/2024-01-29-new-ferretdb-v119-release.md

```diff
@@ -0,0 +1,62 @@
+---
+slug: new-ferretdb-v119-release
+title: FerretDB releases v1.19.0
+authors: [alex]
+description: >
+  We just released FerretDB v1.19.0, which includes support for creating an index on nested fields in SQLite and some important bug fixes.
+image: /img/blog/ferretdb-v1.19.0.jpg
+tags: [release]
+---
+
+![FerretDB releases v1.19.0](/img/blog/ferretdb-v1.19.0.jpg)
+
+We just released FerretDB v1.19.0, which includes support for creating an index on nested fields in SQLite and some important bug fixes.
+
+In recent weeks, we've been working on enabling support for authentication.
+That way, [FerretDB](https://www.ferretdb.com/) can manage users by itself and not rely on the backend to do so.
```

Comment:
Authentication is already exist and enabled: https://docs.ferretdb.io/security/authentication/